service/endpoint_lifecycle_subscriber.hh

```diff
@@ -66,7 +67,7 @@ public:
     future<> unregister_subscriber(endpoint_lifecycle_subscriber* subscriber) noexcept;
 
     future<> notify_down(gms::inet_address endpoint);
-    future<> notify_left(gms::inet_address endpoint);
+    future<> notify_left(gms::inet_address endpoint, locator::host_id hid);
```

Comment:
It would be better to convert all the callbacks to use host ID exclusively, but I understand it's not practical. It's okay to have this redundant call convention, but let's plan to convert everything later.